ansible/roles/app/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+- name: Test if app_dev.php.dist exists
+  command: test -f /var/www/{{ server.name }}/app_dev.php.dist
```

Comment:
please place this file inside that ansible/roles/app/files folder, as that is where it belongs - and only copy it when in dev mode :tongue: